pycontw/pycon.tw
# Feature/788 community track venue gallery

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [ * ] **New feature**

## Description
This pr is for issue #788 


## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to '/en-us/sponsor/community-tracks/' and '/zh-hant/sponsor/community-tracks/'
2. Click on gallery and check pic is right

## Expected behavior
Every gallery is work and picture is right

## Related Issue
close #788 

## More Information

![vueue-hant](https://user-images.githubusercontent.com/14121801/88598373-bd244300-d09b-11ea-8538-e448daae0aaa.png)



**Additional context**
Add any other context about the problem here. You may also want to refer
to [how to wirte the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)
